### PR TITLE
Create executable visual studio project for NativeTestSuiteBinarySpec

### PR DESCRIPTION
**Pull request by donalmacc**
Any of the checked boxes below indicate that I took action:

- [X] Reviewed the [Contribution Guidelines](https://github.com/gradle/gradle/blob/master/.github/CONTRIBUTING.md).
- [X] Signed the [Gradle CLA](http://gradle.org/contributor-license-agreement/).
- [X] Ensured that basic checks pass: `./gradlew quickCheck`

For all non-trivial changes that modify the behavior or public API:

- [ ] Before submitting a pull request, I started a discussion on the [Gradle developer list](https://groups.google.com/forum/#!forum/gradle-dev),
      the [forum](https://discuss.gradle.org/) or can reference a [JIRA issue](https://issues.gradle.org/secure/Dashboard.jspa).
- [ ] I considered writing a design document. A design document can be
brief but explains the use case or problem you are trying to solve,
touches on the planned implementation approach as well as the test cases
that verify the behavior. Optimally, design documents should be submitted
as a separate pull request. [Samples](https://github.com/gradle/gradle/tree/master/design-docs)
can be found in the Gradle GitHub repository. Please let us know if you need help with
creating the design document. We are happy to help!
- [ ] The pull request contains an appropriate level of unit and integration
test coverage to verify the behavior. Before submitting the pull request
I ran a build on your local machine via the command
`./gradlew quickCheck <impacted-subproject>:check`.
- [ ] The pull request updates the Gradle documentation like user guide,
DSL reference and Javadocs where applicable.

This is a fix for the issue at https://github.com/gradle/gradle/issues/844. I'm not sure what the required extra work from me is for such a trivial change.

### CODE REVIEW

File: subprojects/ide-native/src/main/groovy/org/gradle/ide/visualstudio/internal/VisualStudioProjectRegistry.java [hunk 1]
```diff
@@ -52,7 +53,7 @@ public VisualStudioProjectConfiguration addProjectConfiguration(NativeBinarySpec
 
     private VisualStudioProjectConfiguration createVisualStudioProjectConfiguration(NativeBinarySpec nativeBinary, DefaultVisualStudioProject project, String configuration, String platform) {
         Class<? extends VisualStudioProjectConfiguration> type =
-                nativeBinary instanceof NativeExecutableBinarySpec ? ExecutableVisualStudioProjectConfiguration.class : VisualStudioProjectConfiguration.class;
+            (nativeBinary instanceof NativeExecutableBinarySpec  || nativeBinary instanceof NativeTestSuiteBinarySpec)? ExecutableVisualStudioProjectConfiguration.class : VisualStudioProjectConfiguration.class;
```
Comment (eriwen) [inline subprojects/ide-native/src/main/groovy/org/gradle/ide/visualstudio/internal/VisualStudioProjectRegistry.java]: Please make white spacing consistent on this line with the rest of the project.

**Comment by eriwen** [comment]
@lacasseio When you have a moment, would you please give this a quick check?

**Comment by lacasseio** [comment]
@eriwen Everything looks good. I could merge it for you, however, I cannot find the CLA for @donalmacc. Did he sign it?

**Comment by donalmacc** [comment]
Hi,
Apologies I've been away on holiday. I'll fix the whitespace, and the CLA has been signed since you looked.

**Comment by lacasseio** [comment]
Thanks @donalmacc, everything is in order, I will merge this.

**Pull request merged**